themis/_index.md

```diff
@@ -11,25 +11,67 @@ Menu: "main"
 _What Themis is and why it's awesome_
 
 
-## What is Themis
+## What Themis is
+
+Themis is a cross-platform high-level cryptographic library for mobile, web and server platforms. Themis solves 90% of typical data protection use cases that are common for most apps. 
+
+Themis provides ready-made building blocks _(["crypto-systems"](#crypto-systems))_, which simplify usage of core cryptographic security operations.
```

Comment:
```suggestion
Themis provides ready-made building blocks _(["cryptosystems"](#cryptosystems))_ which simplify usage of core cryptographic security operations.
```

I'd spell it “cryptosystem”, as a single word, like we do in `themis/crypto-theory/crypto-systems/_index.md`. Or change it to dashed spelling elsewhere, but the former seems to be more widespread.

Also, I hate to be this pointing out person, but commas: you don't need so many. Restrictive clauses typically are not set off.